style.css

```diff
@@ -942,6 +947,27 @@ body {
     height: 410px;
     top: 65%;
   }
+
+  .hamburger_button {
+    position: absolute;
+    left: 306px;
+    top: 40px;
+    background: none;
+    color: none;
+    z-index: 3;
+    width: 50px;
+    height: 50px;
+  }
+
+  .menu_link {
+    font-size: 45px;
+    height: 50px;
+    padding-bottom: 50px;
+    width: 100vh;
+    border-bottom: solid;
+    border-width: 1px;
+    border-color: #6f6c6b;
+  }
```

Comment:
- [ ] Kindly make the menu links placed at the top, not in the middle like you did to match the given design.
- [ ] Please add the correct font size and margin values for the mobile menu links, Because now it's too large and miss placed.
- [ ] Please remove the button border style around the hamburger icon. When the button is clicked and the mobile menu appeared it's supposed to be a close (X) icon, not a hamburger icon, So please fix that.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/208545237-416fb971-24fe-4fdb-88bc-eef581fb8279.png) | ![image](https://user-images.githubusercontent.com/68796232/208545148-3789cc30-fc20-42e2-b7ec-95513c755a40.png)  |